web/services/checks.go

```diff
@@ -117,23 +107,37 @@ func (c *checksService) CreateChecksCatalogEntry(check *models.Check) error {
 		return err
 	}
 
-	checkRaw := models.CheckRaw{ID: check.ID, Payload: checkJson}
+	checkEntity := entities.Check{ID: check.ID, Payload: checkJson}
 	result := c.db.Clauses(clause.OnConflict{
 		UpdateAll: true,
-	}).Create(&checkRaw)
+	}).Create(&checkEntity)
 
 	return result.Error
 }
 
 func (c *checksService) CreateChecksCatalog(checkList models.ChecksCatalog) error {
+
+	var checkEntityList entities.CheckList
 	for _, check := range checkList {
-		result := c.CreateChecksCatalogEntry(check)
-		if result != nil {
-			return result
+		// Unfortunately, with the current folders structure, we cannot have model to entity funcs
```

Comment:
Actually we should never have a `.ToEntity` method in a model, that would break [dependency rule](https://blog.cleancoder.com/uncle-bob/2012/08/13/the-clean-architecture.html) where higher level components - mdels in our case - do not depend/mention lower level components - entity, which is more of an infrastructural concern.

Maybe a function `toEntity(Model) Entity` unrelated from the model itself could be ok, but I am not completely sure :sweat_smile: 

I would remove this comment.